src/content/tutorials/fleet-management/cluster-management/_index.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Clusters
+title: Cluster Management
 description: Management of workload clusters across different regions and cloud providers.
```

Comment:
From this vague description, I'd guess this will be an article about multi-cloud? If so, the title and description should be made more explicit (doesn't need to be in this PR).